next/src/app/co-robie/page.tsx

```diff
@@ -0,0 +1,206 @@
+        key={i}
+      />
+    ),
+    process: (
+      <ProcessComponent
+        data={component as Process}
+        key={i}
+      />
+    ),
+    tilesSmall: (
+      <SmallTilesComponent
+        data={component as Tiles}
+        key={i}
+      />
+    ),
+  });
+
+  return (
+    <>
+      {content.map((component, i) => {
+        return mappedComponents(component, i)[component._type] as JSX.Element;
+      })}
+    </>
+  );
+}
+
+async function getMetadata() {
+  const { page } = await sanityFetch<MyWorkPage>({
+    query: /* groq */ `{
+      "page": *[_id=="MyWorkPage"] {
+      seo {
+       ...,
+       og_Img {
+         asset -> {
+           url
+         }
+       }
+     }
+   }
+    }`,
+  });
+  return page;
+}
+
+async function getData() {
+  const { page } = await sanityFetch<MyWorkPage>({
+    query: /* groq */ `{
+      "page": *[_id=="MyWorkPage"][0]{
+        content[] {
+          ...,
+          list[] {
+            ...,
+            image {
+              asset -> {
+                altText,
+                url,
+                metadata {
+                  lqip,
+                  dimensions {
+                    height,
+                    width
+                  }
+                }
+              }
+            }
+          },
+          cards[] {
+            ...,
+            icon {
+              asset -> {
+                altText,
+                url,
+                metadata {
+                  lqip,
+                  dimensions {
+                    height,
+                    width
+                  }
+                }
+              }
+            }
+          },
+          image {
+            asset -> {
+              altText,
+              url,
+              metadata {
+                lqip,
+                dimensions {
+                  height,
+                  width
+                }
+              }
+            }
+          },
+          faq[] {
+            ...,
+            image {
+              asset -> {
+                altText,
+                url,
+                metadata {
+                  lqip,
+                  dimensions {
+                    height,
+                    width
+                  }
+                }
+              }
+            }
+          },
+          proceses[] {
+            ...,
+            image {
+              asset -> {
+                altText,
+                url,
+                metadata {
+                  lqip,
+                  dimensions {
+                    height,
+                    width
+                  }
+                }
+              }
+            }
+          },
+          card {
+            ...,
+            image {
+              asset -> {
+                altText,
+                url,
+                metadata {
+                  lqip,
+                  dimensions {
+                    height,
+                    width
+                  }
+                }
+              }
+              }
```

Comment:
prettier is not working properly

Reply:
Prettier does not support groq and there are no plugins currently available